Week1/homework/allQueries.js

```diff
@@ -0,0 +1,173 @@
+var mysql = require('mysql');
```

Comment:
Hi @AhmadAbdulhaleem, question: how are you requiring your node packages? I don't see any package.json / package-lock.json in this change